assets/src/styles/blocks/CarouselHeader/CarouselHeaderEditorStyle.scss

```diff
@@ -99,3 +113,9 @@
     margin: 0 !important;
   }
 }
+
+@include x-large-and-up {
```

Comment:
Had to add this to avoid having the text inputs on top of the left arrow in the backend